litentry/primitives/src/validation_data.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_runtime::{traits::ConstU32, BoundedVec};
+// use sp_runtime::{traits::ConstU32, BoundedVec, MultiSignature};
+
+pub type MaxStringLength = ConstU32<64>;
+pub type ValidationString = BoundedVec<u8, MaxStringLength>;
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct TwitterValidationData {
```

Comment:
Do we need the whole URL/the user handle as well? Or can you retrieve it by tweet_id? I know tweet_id is globally unique, but not sure about the complexity of querying with that.

Reply:
Only tweet_id is required to retrieve tweets via the API. This is simple.
like:
```bash
curl "https://api.twitter.com/2/tweets?ids=1571829863862116352&expansions=author_id" -H "Authorization: Bearer $BEARER_TOKEN"
```